code/mob/living/critter/small_animal.dm

```diff
@@ -4363,14 +4377,20 @@ var/list/mob_bird_species = list("smallowl" = /mob/living/critter/small_animal/b
 	void_mindswappable = FALSE
 	player_can_spawn_with_pet = FALSE
 	has_genes = FALSE
+	shiny_chance = 0
 
 	New()
 		..()
-		src.real_name = "[pick_string("mentor_mice_prefixes.txt", "mentor_mouse_prefix")] [src.name]"
+		if(src.is_shiny)
+			src.icon_state = "mouse-large-shiny"
+			src.icon_state_dead = "mouse-large-shiny-dead"
+			src.icon_state_exclaim = "mouse-large-shiny-exclaim"
+		else
+			src.real_name = "[pick_string("mentor_mice_prefixes.txt", "mentor_mouse_prefix")] [src.name]"
```

Comment:
name?

Reply:
Could you provide more detail as to the issue here? The "shiny" prefix is assigned in the parent proc and so doesn't need to be added here.